Sources/SlidableImage.swift

```diff
@@ -57,23 +78,49 @@ open class SlidableImage: UIView {
   ///
   /// - Parameter maskLocation: Position of slider
   open func updateMask(location maskLocation: CGFloat) {
-    let maskRectPath = UIBezierPath(rect: CGRect(x: bounds.minX,
+    var maskRectPath: UIBezierPath
+    let mask = CAShapeLayer()
+    switch slideDirection {
+      case .left:
+        maskRectPath = UIBezierPath(rect: CGRect(x: maskLocation,
```

Comment:
Just CGRect declaration should be in case, it would be clearer what is really switching